docker/docker-compose.transport-ci.yml

```diff
@@ -0,0 +1,18 @@
+version: "3"
+services:
+  trezor-user-env-unix:
+    image: registry.gitlab.com/satoshilabs/trezor/trezor-user-env/trezor-user-env
+    environment:
+      - SDL_VIDEODRIVER=dummy
+      - XDG_RUNTIME_DIR=/var/tmp
+    network_mode: bridge  # this makes docker reuse existing networks
+  test-run:
+    build:
+      context: .
+      dockerfile: ./transport/Dockerfile
```

Comment:
why not use cypress image as in our e2e tests?

Reply:
Ideally I would like to ditch this file in case I can use "./docker/run-next-to-tuenv.sh` approach. 
Cypress image does not feel right as it has way more dependencies than I need. All I need is actually available in the base image.